src/pallavi/Assignment_15/UserAccountInfo.java

```diff
@@ -0,0 +1,183 @@
+package pallavi.Assignment_15;
+
+import java.util.Scanner;
+
+public class UserAccountInfo {
+	int iBalance = 50000;
+	int hBalance = 30000;
+	int bBalance = 60000;
+	int sBalance = 80000;
+
+	int iCreditCount = 0, hCreditCount = 0, bCreditCount = 0, sCreditCount = 0;
+	int iDebitCount = 0, hDebitCount = 0, bDebitCount = 0, sDebitCount = 0;
+
+	void totalDebitandCreditCount() {
+		int totalDebit = iDebitCount + hDebitCount + bDebitCount + sDebitCount;
+		System.out.println("Total debited transactions count are :" + totalDebit + " times");
+		int totalCredit = iCreditCount + hCreditCount + bCreditCount + sCreditCount;
+		System.out.println("Total credit operation :" + totalCredit + " times");
+	}
+
+	void indivisualCreditOperation() {
+		System.out.println(" ");
+		System.out.println("------------You will get total Credit operations indivisually---------------");
+		System.out.println("ICICI bank Credit Operation : " + iCreditCount + " times");
+		System.out.println(" HDFC bank Credit Operation :" + hCreditCount + " times");
+		System.out.println("SBI bank Credit Operation :" + sCreditCount + " times");
+		System.out.println("BOB bank Credit Operation :" + bCreditCount + " times");
+	}
+
+	void indivisualDebitOperation() {
+		System.out.println(" ");
+		System.out.println("------------You will get total Debit operations indivisually---------------");
+		System.out.println("ICICI bank Debit Operation : " + iDebitCount + " times");
+		System.out.println(" HDFC bank Debit Operation :" + hDebitCount + " times");
+		System.out.println("SBI bank Debit Operation :" + sDebitCount + " times");
+		System.out.println("BOB bank Debit Operation :" + bDebitCount + " times");
+	}
+
+	void operation(String op, String bank, int amount) {
+		switch (op) {
+		case "debit":
+			switch (bank) {
+			case "ICICI":
+				if (iBalance >= amount) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method